uber/model_checks.py

```diff
@@ -74,7 +74,7 @@ def group_money(group):
         amount_refunded = int(float(group.amount_refunded))
         if amount_refunded < 0:
             return 'Amount Refunded must be positive'
-        elif amount_refunded > amount_paid:
+        elif amount_refunded > group.amount_paid:
```

Comment:
Upon further review, this should be:
```
elif amount_refunded > amount:
```

Because amount is a normalized `int` set earlier in the function.

Honestly, I'd change:
```
amount = int(float(group.amount_paid))
```

To:
```
amount_paid = int(float(group.amount_paid))
```